src/pipe.toit

```diff
@@ -165,9 +165,20 @@ To avoid zombies you must either give the child process to either
   `dont_wait_for` or `wait_for`.
 Optionally you can pass pipes that should be passed to the
   child process as open file descriptors 3 and/or 4.
+Optionally, $environment variables can be passed as a map.
+  Keys in the map should be strings, and values should be strings or null,
+  where null indicates that the variable should be unset in the child
+  process.
+Note that if you override the PATH environment variable, but set the $use_path
+  flag, the new value of PATH will be used to find the executable.
 */
-fork use_path stdin stdout stderr --file_descriptor_3/OpenPipe?=null --file_descriptor_4/OpenPipe?=null command arguments -> List:
+fork use_path stdin stdout stderr --environment/Map?=null --file_descriptor_3/OpenPipe?=null --file_descriptor_4/OpenPipe?=null command arguments -> List:
   result := List 4
+  flat_environment := environment ? (Array_ environment.size * 2) : null
+  index := 0
+  if environment: environment.do: | key value |
+    flat_environment[index++] = key.stringify
+    flat_environment[index++] = value ? value.stringify : null
```

Comment:
What if `value` is 'false' ?
or do you only care for `null` here?
if it's only `null` then you might as well write `value and value.stringify`.

Reply:
changed to: (value == null) ? null : value.stringify

which will pass false as a string, not as a bool.